session_0/item.rb

```diff
@@ -41,5 +51,12 @@ def discounted?
   # Note: If there are no items for category, stock price for category
   # should be zero.
   def self.stock_price_by_category(items)
+    sp = Hash.new 
+    sp = {
```

Comment:
Woah!

That's definitely a new approach - do you think it would still work if there are 100 categories?